crates/bevy_transform/src/components/transform.rs

```diff
@@ -48,6 +52,8 @@ impl Transform {
     /// Creates a new [`Transform`] at the position `(x, y, z)`. In 2d, the `z` component
     /// is used for z-ordering elements: higher `z`-value will be in front of lower
     /// `z`-value.
+    /// 
+    /// Example: [translations](https://github.com/bevyengine/bevy/blob/latest/examples/transforms/translation.rs)
```

Comment:
I think that this link would be a better fit on the field.

Reply:
gotcha! changed the location to the translate field